### DISCUSSION TITLE
为什么无法修改基础模型

### DISCUSSION THREAD

**Discussion by robinwang25** [lllyasviel/Fooocus#1334] [General]
![1702299497400](https://github.com/lllyasviel/Fooocus/assets/141383987/f10a4a1e-2b55-4b1b-bfe8-287d154e8a80)
上周用还好好的，这周用就无法选择莫型了，只要是带收缩框选项的，都无法选择，这是为什么？

**Comment by metercai** [comment]
你这个看着是个修订版，不是原生版本。换原生版，或者这个：https://github.com/metercai/SimpleSDXL  支持中文，新增了很多实用功能：中英混编提示词，历史图片检索，提参重生，自定义预置包等
